### ISSUE TITLE
Build error on Windows with Qt 5.9.2

### ISSUE THREAD

**Issue by Xaxetrov** [commontk/CTK#851]
Hello,
I am trying to build CTK from scratch to update the version of QtTesting it uses, do some tests end create a pull requests.

These steps have been tried for both ming32-make and visual studio 15 (2017) generators:
- I cloned this repo
- I configured and generated using cmake-gui, setting:
  - CTK_QT_VERSION to 5
  - Qt5_DIR to ...\Qt\Qt5.9.2\5.9.2\\_chosen_generator_\lib\cmake\Qt5. The configure then correctly sets other related qt5 cache variables
  - CMAKE_PREFIX_PATH to .../Qt/Qt5.9.2/5.9.2/_chosen_generator_/lib/cmake
- I got no error and everything seemed fine.
- I ran the chosen generator

For both generators, 'QTTesting' and 'CTKData' steps go well but I get the same error at build time in 'CTK':
...\CTK\Libs\Widgets\ctkActionsWidget.cpp:22:10: fatal error: QAction: No such file or directory #include <QAction>

Is there something more to do ?

@lassoan @jcfr 
Also tagging you @finetjul so that you can follow.

**Comment by lassoan** [comment]
Can you find the missed file on your computer (on mine it is: `c:\Qt\5.10.0\msvc2015_64\include\QtWidgets\QAction`).

Do you pass Qt5_DIR before the first configuration? It may be too late to set it after once you have already attempted to configure the project.

Do you have "C:\Qt\5.10.0\msvc2015_64\include\QtWidgets" listed in CTKWidget's project properties / Configuration Properties / C/C++ / General / Additional Include Directories?

Do you have correct path (Qt5Widgets_DIR:PATH=C:/Qt/5.10.0/msvc2015_64/lib/cmake/Qt5Widgets) in CTK-build\CMakeCache.txt and in CTK-build\CTK-build\CMakeCache.txt?

**Comment by Xaxetrov** [comment]
Hi, I will answer to your four paragraphs one by one.

- Yes the "missing" files are in my file system.
- Yes I tried pre-filling CTK_QT_VERSION and Qt5_DIR.
- No I don't but I did not activate CTK_ENABLE_Widgets nor CTK_LIB_WIDGETS though.
- I do not have it in the outer-build CMakeCache but I do have it in the inner-build CMakeCache.

I am going to try switching some CMake options on to see the difference.
Thanks for your help.
